src/insns/load_tag_perms.adoc

```diff
@@ -0,0 +1,10 @@
+Resulting value of `cd`::
+The tag value written to `cd` is 0 if the tag of the memory location loaded is
+0 or the authorizing capability (<<ddc>> or `cs1`) does not grant <<c_perm>>.
++
+If the authorizing capability does not grant <<lm_perm>>, and the tag of `cd` is 1 and `cd` is not sealed, then and implicit <<ACPERM>> clearing <<w_perm>> and <<lm_perm>> is performed to obtain the final permissions on `cd`.
+
+NOTE: Missing <<lm_perm>> does not affect untagged values since this could result in surprising bit patterns when copying non-capability data.
+Similarly, sealed capabilities are not modified as they are not directly derefenceable.
+
+NOTE: The implicit <<ACPERM>> does not introduce a new dependency on the loaded data since the actual masking of permissions is only needed once the loaded capability is dereferenced or the metadata bits are inspected using <<GCPERM>> or <<GCHI>>.
```

Comment:
I disagree with this statement. The ACPERM does introduce a dependency on the loaded data, but it seems that there are microarchitectural tricks that could be implemented to defer the operation a little.

Reply:
yes - the point is that the transformation can be deferred until the data is consumed, it doens't need to be done with the load data is produced. This could be clearer.

Reply:
Agreed, will try to rephrase this